docs/source/tutorial.rst

```diff
@@ -141,7 +172,12 @@ X byproduct, node = 7, domain = [6]
 Z byproduct, node = 7, domain = [5]
 Z byproduct, node = 3, domain = [5]
 
+.. figure:: ./../imgs/pattern_visualization_2.png
+    :scale: 60 %
+    :alt: visualization of optimized pattern
+
 
+As we see below, we can significantly optimize this for better simulation performance and even operations in quantum hardware.
```

Comment:
can you change this line to 
`As we see below, we can simplify and optimize the pattern by calling various methods of `~graphix.pattern.Pattern` class.

Reply:
added to PR